glossary.yml

```
@@ -6060,8 +6060,8 @@
   ja:
     term: "変数 (プログラム)"
     def: >
-      プログラム内にデータが結合するの名前です。変数の値は定義したの後で代再入出来のこと。
-      変数は英語で「variable」(ヴァリアブル)と呼んでいのこと。
+      プログラム内で使用される、データが関連付けられる総称。変数の値は定義した後で変更が可能。
```

We need an object before '総称', e.g., 箱 or 値. If we add extra information like '変数は英語で「variable」(ヴァリアブル)という', it may be okay to use 'a box' there. Anyhow, I will ask others' thoughts on Slack and go from there!

---

Good idea. I'm open to a non-literal translation if it aids understanding. In this case, 箱(box) is more appropriate that 値 (value) I think. So the "variable" is a the name of the box which contains the data or value.

```r
name <- "data"
```

On further reading, I agree that 総称 is better than 名前.

```suggestion
      プログラム内で使用される、データが関連付けられる箱の総称。変数の値は定義した後で変更が可能。
```

---

Probably we can add "のようなもの" after "箱" so that it is clearer that this is a sort of abstract expression rather than concrete thing?
```suggestion
      プログラム内で使用される、データが関連付けられる箱のようなものの総称。変数の値は定義した後で変更が可能。
```